src/main/java/io/jenkins/plugins/report/jtreg/main/comparator/Arguments.java

```diff
@@ -7,85 +7,72 @@ public static Options parse(String[] arguments) {
 
         if (arguments.length >= 1) {
             for (int i = 0; i < arguments.length; i++) {
-                switch (arguments[i].toLowerCase()) {
-                    case "-h":
-                    case "--help":
+                String currentArgument = arguments[i];
+
+                // delete all leading - characters
+                while (currentArgument.charAt(0) == '-') {
+                    currentArgument = currentArgument.substring(1);
+                }
```

Comment:
thanx for this :)

Reply:
however... you should enbsure that there is at least one -.;  So something like argument.repalceall("^-+","-") does itts job. Of course then you will need to match -h/-help .... IU would storngly advice to do that. the `-` is the only thing diferencing argument type form argument itself